inventory-generation/notifications/files/hosts

```diff
@@ -0,0 +1,3 @@
+[tower-management-host]
+localhost ansible_connection=local
```

Comment:
I'd suggest removing the `ansible_connection` from here since you have the `host_vars/localhost.yml` file with the same configuration below

Reply:
The `host_vars/localhost.yml` is used by the inventory-generation playbook for notifications.

This file is moved into the generated inventory by the `copy` module to be used in the next stage when the inventory is consumed by Tower.